cluster-autoscaler/cloudprovider/mcm/mcm_manager.go

```diff
@@ -697,6 +697,16 @@ func validateNodeTemplate(nodeTemplateAttributes *v1alpha1.NodeTemplate) error {
 	return nil
 }
 
+// GetMachineDeploymentSpec returns the machine deployment spec of the provided machine deployment name
+func (m *McmManager) GetMachineDeploymentSpec(machineDeploymentName string) (*v1alpha1.MachineDeployment, error) {
```

Comment:
Can you rename this to `GetMachineDeploymentAnnotations` and return just the annotations? We don't need the entire machine deployment.